Gaijinizer patch/Map122.txt

```diff
@@ -461,63 +481,61 @@ dog?#MTLed
 だけど、代わりに自由気ままに暴れて、喰ってやった。
  隙を見せた人間から、次々とね…
 > CONTEXT: Map122/events/5/pages/1/196/Dialogue/enemy14
-Well, I did cooperate with the "experiment".
-But instead, I went wild and ran wild.
-From people who showed a chance, one after
-another ...#MTLed
+Well, I did cooperate with their experiments,
+but in exchange, they let me run rampant and eat
+whoever I wanted. Anyone who showed me the
+slightest weakness, I hate one by one...#RonIffy
 > END STRING
```

Comment:
Minor rewording:
>_**I cooperated with their experiments, but in
exchange they let me run rampant and eat whoever
I wanted. Anyone who showed me the slightest bit
of weakness was eaten, one by one...**_